### PR TITLE
Update AMI of container and bastion instances to 2.0.20210902

### PR DESCRIPTION
**Pull request by essa**
This PR will update the ami of container instances as following page.

- [Amazon ECS\-optimized AMIs \- Amazon Elastic Container Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-optimized_AMI.html)

And it also updates ami for bastion image.

- https://ap-northeast-1.console.aws.amazon.com/systems-manager/parameters/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2/description?region=ap-northeast-1

And it also upgrades puma and addressable (octokit) to fix vulnerablitity.

### CODE REVIEW

**Comment by degicat** [comment]
@Yuuki77 can you help us review this PR, please?

File: lib/barcelona/network/bastion_builder.rb [hunk 1]
```diff
@@ -6,21 +6,21 @@ class BastionBuilder < CloudFormation::Builder
...
-        "eu-west-3"      => "ami-0d49cec198762b78c",
-        "eu-central-1"      => "ami-0453cb7b5f2b7fca2",
-        "ap-northeast-1"      => "ami-09ebacdc178ae23b7",
-        "ap-northeast-2"      => "ami-0a0de518b1fc4524c",
-        "ap-southeast-1"      => "ami-0f511ead81ccde020",
-        "ap-southeast-2"      => "ami-0aab712d6363da7f9",
-        "ca-central-1"      => "ami-02f84cf47c23f1769",
-        "ap-south-1"      => "ami-04db49c0fb2215364",
-        "sa-east-1"      => "ami-0f8243a5175208e08",
+                "us-east-1"      => "ami-087c17d1fe0178315",
```
Comment (Yuuki77) [inline lib/barcelona/network/bastion_builder.rb:9]: unexpected tab 👁️

**Review by Yuuki77 — COMMENTED** [review]
I confirmed that all values are set correctly
one indentation issue but other than that looks good to me 😄

File: lib/barcelona/network/bastion_builder.rb (hunk 1 above)
Comment (essa) [reply lib/barcelona/network/bastion_builder.rb]: Good catch! Thanks!
I fixed it.

**Pull request merged**